public/ubuntu.dockerfile

```diff
@@ -49,13 +49,6 @@ RUN export CMK_VER=3.31.7 \
   && wget -qO- "https://github.com/Kitware/CMake/${CMK_DL}" \
   | tar --strip-components=1 -xz -C /usr/local/ \
   && unset CMK_DL && unset CMK_VER
-# Dockerfile.vim
```

Comment:
I guess you didn't see a good reason for this? Too bad, because I have found it very useful